admin-ui/src/lib/admin-api.ts

```
@@ -155,14 +161,20 @@ async function fetchAdminJson<T>(path: string, init?: RequestInit): Promise<T> {
       // ignore JSON parsing errors and keep raw text
     }
 
+    if (res.status === 401 || res.status === 403) {
+      if (typeof window !== "undefined") {
+        window.dispatchEvent(new Event(ADMIN_TOKEN_REQUIRED_EVENT))
+      }
```

ESLint

[error] Unexpected `if` as the only statement in a `if` block without `else`.

(unicorn/no-lonely-if)